src/SortableList/index.js

```diff
@@ -42,24 +42,18 @@ class SortableList extends React.PureComponent {
     const rowIndex = this.findHighestUpdatedRow(this.props.list.rows, prevProps.list.rows);
     if (rowIndex !== null) {
       this.cache.clearAll();
-      if (this._list) this._list.wrappedInstance.recomputeRowHeights(rowIndex + 1);
```

Comment:
`recomputeRowHeights ` recomputes the row heights and offsets after the specified index. I believe I mistakenly thought it was doing the given index and tried to manually choose the next row.